03-seeking-help.Rmd

```diff
@@ -124,6 +124,11 @@ your issue.
 > What is the difference between the `sep` and `collapse` arguments?
 > 
 
+> ## Challenge 3 {.challenge}
+> Use help to find out which function and with what parameters should you use
```

Comment:
I think you should keep the hint here especially as `?read.csv` redirects to `?read.table`